sgt-back-end/index.js

```diff
@@ -0,0 +1,152 @@
+const pg = require('pg');
+const express = require('express');
+const app = express();
+
+app.use(express.json());
+
+// only create ONE pool for your whole server
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades', (req, res) => {
+
+  const sql = `
+    select "gradeId",
+           "name",
+           "course",
+           "score",
+           "createdAt"
+      from "grades"
+      `;
+  db.query(sql)
+    .then(result => {
+      const grade = result;
+
+      res.json(grade);
```

Comment:
This will send back more data than we want, we just want to send back the array of results so we should be using `result.rows`
```JAVASCRIPT
const grades = result.rows;

res.json(grades);
```